doc/src/build/fullnode.md

```diff
@@ -288,3 +288,119 @@ If you followed the instructions for [Building from Source](#building-from-sourc
     ```
 
 Your Full node starts on: `http://127.0.0.1:9000`.
+
+# Pruning 
+
+## Object Pruning
+
+Object pruning involves the removal of older object versions from the fullnode db. 
+New object versions are added to the db during transaction execution (making older versions ready for garbage collection) and can become a 
+problem if not pruned, as it can slow down the performance of the db and take up too much disk space. 
+Object pruning works in the background by tailing checkpoint stream, identifying each
+checkpoint which contains object versions which are now eligible for pruning.
+
+A sui node can enable object pruning by adding this `authority-store-pruning-config` config:
+```yaml
+authority-store-pruning-config:
+  # Number of epoch dbs to keep 
+  # Not relevant for object pruning
+  num-latest-epoch-dbs-to-retain: 3
+  # How often the epoch db pruning task should run
+  # Not relevant for object pruning
+  epoch-db-pruning-period-secs: 3600
+  # Number of latest epochs before which object pruning is allowed to run.
+  # When this is 0, pruner is allowed to prune old object versions as soon
+  # as possible. This is what we also call aggressive pruning and is the most effective
+  # garbage collection method which results in the samllest disk usage possible. 
+  # This is recommended setting for validator nodes as older object versions aren't
+  # needed for transaction execution.
```

Comment:
```suggestion
  # When set to 0, Sui prunes old object versions as soon
  # as possible. This is also called *aggressive pruning*, and results in the most effective
  # garbage collection method with lowest disk usage possible. 
  # This is the recommended setting for Sui Validator nodes since older object versions aren't
  # necessary to execute transactions.
```